# Define the goal
sensor_msgs/PointCloud2 pointcloud_scene
float64 voxel_size
---
# Define the result
geometry_msgs/PoseStamped target_grasp_pose
---
# Define the feedback
